lib/rspec/expectations/configuration.rb

```diff
@@ -185,6 +186,21 @@ def on_potential_false_positives=(behavior)
         @on_potential_false_positives = behavior
       end
 
+
```

Comment:
This extra blank line is failing rubocop.

Reply:
Oh, I'm sorry about that. Fixed